packages/core/EventManager.ts

```diff
@@ -294,6 +294,13 @@ export default class EventManager {
   }
 
   public async updateCalendarAttendees(event: CalendarEvent, booking: PartialBooking) {
+    const attendees = booking.attendees.map((attendee) => {
+      return {
+        email: attendee.email,
+        name: attendee.name,
+      };
+    });
+    event.attendees = attendees;
```

Comment:
This was the problem, it looks like code that copies attendees to event was missing here.

Reply:
Is there any situation where event.attendees would be set? (And this would overwrite that existing value?)